ctapipe/tools/muon_reconstruction.py

```diff
@@ -174,13 +194,7 @@ def process_telescope_event(self, event_index, tel_id, dl1):
             self.log.debug(f'Skipping event {event_id}-{tel_id}: Ring fit did not succeed')
             return
 
-        # add ring containment, not filled in fit
-        ring.containment = ring_containment(
-            ring.radius,
-            fov_radius,
-            ring.center_x,
-            ring.center_y,
-        )
+        parameters = self.calculate_muon_parameters(tel_id, image, clean_mask, ring)
 
         # intensity_fitter does not support a mask yet, set ignored pixels to 0
```

Comment:
This will not work exactly as desired, the pixels will enter the fit all the same, even with 0 values. MuonIntensityFitter should support masks

Reply:
Yes, agreed. Setting them to zero will bias the computation.